src/engraving/dom/tuplet.h

```diff
@@ -84,8 +85,10 @@ class Tuplet final : public DurationElement
     void setBracketType(TupletBracketType val) { m_bracketType = val; }
     bool hasBracket() const { return m_hasBracket; }
     void setHasBracket(bool b) { m_hasBracket = b; }
-    Millimetre bracketWidth() const { return m_bracketWidth; }
-    void setBracketWidth(Millimetre s) { m_bracketWidth = s; }
+    Spatium bracketWidth() const { return m_bracketWidth; }
+    void setBracketWidth(Spatium s) { m_bracketWidth = s; }
+
+    double point(Spatium sp) const override { return sp.val() * score()->style().spatium(); }
```

Comment:
Same as above

Reply:
Potentially the same spatium/chord mag confusion.  If so, we won't need these overrides at all.